.travis.yml

```diff
@@ -12,8 +12,8 @@ env:
   - VERSION=1.11 PYTHON_VERSION=3.6 PG_MAJOR=10 PG_VERSION=10.4-2.pgdg90+1 VARIANT=slim
   - VERSION=1.11 PYTHON_VERSION=2.7 PG_MAJOR=10 PG_VERSION=10.4-r0 VARIANT=alpine
   - VERSION=1.11 PYTHON_VERSION=3.6 PG_MAJOR=10 PG_VERSION=10.4-r0 VARIANT=alpine
-  - VERSION=2.0 PYTHON_VERSION=3.6 PG_MAJOR=10 PG_VERSION=10.4-2.pgdg90+1 VARIANT=slim
-  - VERSION=2.0 PYTHON_VERSION=3.6 PG_MAJOR=10 PG_VERSION=10.4-r0 VARIANT=alpine
+  - VERSION=2.0 PYTHON_VERSION=3.7 PG_MAJOR=10 PG_VERSION=10.4-2.pgdg90+1 VARIANT=slim
```

Comment:
I think that it would be valuable to keep the 3.6 version alongside 3.7 for Django 2.0. I don't know much about the differences between the two yet, but it is likely that folks will continue to use 3.6 internally, so we'll want to keep updating it until people can migrate to 3.7.